contrib/firmware_packager/install_dell_bios_exe.py

```diff
@@ -10,7 +10,11 @@
 import tempfile
 import gi
 
-gi.require_version('Fwupd', '2.0')
+try:
+    gi.require_version('Fwupd', '2.0')
+except ValueError:
+    print("Missing gobject-introspection packages.  Try to install gir1.2-fwupd-2.0 or fwupd-devel.")
```

Comment:
I think for Fedora the typelib is in the main fwupd package:

    $ rpm -qf /usr/lib64/girepository-1.0/Fwupd-2.0.typelib
    fwupd-1.5.6-0.705.20210120git.fc33.x86_64

The GIR is indeed in the -devel package, but I don't think you need that for `gi`

Reply:
OK dropped that.  I guess this really only is a Debian/Ubuntu problem then since it's not installed by default there.